spec/services/attribute_asserter_spec.rb

```diff
@@ -327,6 +327,31 @@
         end
       end
 
+      context 'service provider specifies attribute bundle' do
+        before do
+          allow(service_provider.metadata).to receive(:[]).with(:attribute_bundle).
```

Comment:
wouldn't it be easier to just stub the metadata as a hash?

```
# earlier in the file
let(:service_provider) do
    instance_double(
      ServiceProvider,
      issuer: 'http://localhost:3000',
      ial: service_provider_ial,
      default_aal: service_provider_aal,
      liveness_checking_required: false,
      metadata: metadata, # change this line
    )
...

let(:metadata) do
 { attribute_bundle: %w[email first_name last_name ssn phone] }
end
```

Reply:
Thanks!